model.py

```
@@ -23,8 +23,8 @@ class LayerNorm(nn.Module):
 
     def __init__(self, ndim, bias):
```

Ruff (0.14.8)

Unused method argument: `ndim`

(ARG002)